src/server.cc

```diff
@@ -1,5 +1,8 @@
+#include "request_handler.h"
+#include "status_count.h"
 #include "server.h"
 #include "http_constants.h"
+#include <vector>
 
```

Comment:
Put #includes (besides server.h and http_constants.h) in server.h. It should work (it's worked for me lol